### PR TITLE
py-bakta: new package

### PR DESCRIPTION
**Pull request by luke-dt**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-bakta/package.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+class PyBakta(PythonPackage):
+    """Bakta: rapid & standardized annotation
+    of bacterial genomes, MAGs & plasmids"""
+
+    homepage = "https://github.com/oschwengers/bakta"
+    pypi = "bakta/bakta-1.5.1.tar.gz"
+
+    maintainers = ["oschwengers"]
+
+    version("1.5.1", sha256="36781612c4eaa99e6e24a00e8ab5b27dadf21c98ae6d16432f3e78c96a4adb5d")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-bakta/package.py:19]: Confirmed sha256.

File: var/spack/repos/builtin/packages/py-bakta/package.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+class PyBakta(PythonPackage):
+    """Bakta: rapid & standardized annotation
+    of bacterial genomes, MAGs & plasmids"""
+
+    homepage = "https://github.com/oschwengers/bakta"
+    pypi = "bakta/bakta-1.5.1.tar.gz"
+
+    maintainers = ["oschwengers"]
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-bakta/package.py:17]: @oschwengers Are you okay with being added as a maintainer of this package?

**Review by tldahlgren — COMMENTED** [review]
Confirmed sha256.

**Comment by tldahlgren** [comment]
Added @adamjstewart since he is the python package guru and chief maintainer.

**Review by adamjstewart — APPROVED** [review]
LGTM, will give @oschwengers a chance to respond before merging.

File: var/spack/repos/builtin/packages/py-bakta/package.py (hunk 2 above)
Comment (oschwengers) [reply var/spack/repos/builtin/packages/py-bakta/package.py:17]: Hi and thanks for asking.
Yes, I'm OK with that. I have no experience with Spack so far, but I'll jump in if I can help with any tool-related questions/issues.

**Review by oschwengers — APPROVED** [review]
Looks good to me. Thanks for the effort and this new option to install/distribute Bakta!

**Pull request merged**